docs/cell_census_schema_0.0.1.md

```diff
@@ -0,0 +1,853 @@
+# CELLxGENE Cell Census Schema 
+
+**Status**: Draft.
+
+**Version**: 0.0.1.
+
+**Last edited**: Dec, 2022.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED" "MAY", and "OPTIONAL" in this document are to be interpreted as described in [BCP 14](https://tools.ietf.org/html/bcp14), [RFC2119](https://www.rfc-editor.org/rfc/rfc2119.txt), and [RFC8174](https://www.rfc-editor.org/rfc/rfc8174.txt) when, and only when, they appear in all capitals, as shown here.
+
+## Cell Census overview
+
+The Cell Census is a versioned container for the single-cell data hosted at [CELLxGENE Discover](https://cellxgene.cziscience.com/). The Cell Census utilizes [SOMA](https://github.com/single-cell-data/TileDB-SOMA) powered by [TileDB](https://tiledb.com/products/tiledb-embedded) for storing, accessing, and efficiently filtering data.
+
+The main goals of the Cell Census are:
+
+1. To allow dataset-independent querying of single-cell data for reproducible analysis by computational biologists via Python and R [APIs](https://github.com/chanzuckerberg/cell-census) that enable on-demand generation of AnnData and Seurat objects respectively.
+1. To enable features of CELLxGENE Discover to access versioned snapshots of its data on cross-dataset features like [Gene Expression](https://cellxgene.cziscience.com/gene-expression). 
+
+To better understand this document the reader should be familiar with the [CELLxGENE dataset schema](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md) and [SOMA](https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md).
+
+## Definitions
+
+The following terms are used throughout this document:
+
+* adata – generic variable name that refers to an [`AnnData`](https://anndata.readthedocs.io/) object.
+* CELLxGENE dataset schema – the data schema for h5ad files served by CELLxGENE Discover, for this Cell Census schema: [CELLxGENE dataset schema version is 3.0.X](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md)
+* census\_obj – the Cell Census root object, a SOMACollection.
+* Cell Census release – a Cell Census deposited in a public bucket and accessible by APIs.
+* tissue – original tissue annotation.
```

Comment:
Is this necessary to define?  If yes, do we also need `cell_type`, `assay`, etc?

Reply:
It's defined because of tissue_general